engine/setup/CMakeLists.txt

```diff
@@ -9,18 +9,18 @@ SET(VEGASETTINGS_SOURCES src/c/setup.cpp
                     ../src/common/common.cpp )                                                                                                        
 
 
-add_executable(vegasettings ${VEGASETTINGS_SOURCES})
+add_executable(vssetup ${VSSETUP_SOURCES})
 
-#find GTK2
-find_package(GTK2 REQUIRED)
-IF(GTK2_FOUND)
-    SET(GTK_LIBS ${GTK2_LIBRARIES})
-    SET(GTK_CFLAGS ${GTK2_DEFINITIONS} -DGTK)
-ELSE(GTK2_FOUND)
-     message("Cant get GTK2 found")
-ENDIF(GTK2_FOUND)
+#find GTK3
+find_package(GTK3 REQUIRED)
```

Comment:
Is this supported by cmake 3.5?

Reply:
Not, sure I'm using Cmake 3.5

Reply:
based on the CI I'd say it's not, meaning we need something to define it.

Reply:
@dmjc thanks for including the related CMake file